client.go

```diff
@@ -73,19 +73,21 @@ type Client interface {
 	LSLBList(ls string) ([]*LoadBalancer, error)
 
 	// Add ACL to entity (PORT_GROUP or LOGICAL_SWITCH)
-	ACLAddEntity(entityType EntityType, entity, direct, match, action string, priority int, external_ids map[string]string, logflag bool, meter string, severity string) (*OvnCommand, error)
+	ACLAddEntity(entityType EntityType, entityUUID, aclName, direct, match, action string, priority int, external_ids map[string]string, logflag bool, meter, severity string) (*OvnCommand, error)
```

Comment:
Why entityUUID is preferred than entityName? I know this has been a big discussion on slack. I replied there as well.

Reply:
I'll respond in slack so we can keep the discussion in one place and then comment back here when there is a decision.